app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>HelloRailsReact</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <link rel="preload" href="http://127.0.0.1:3000/assets/es-module-shims.min-4ca9b3dd5e434131e3bb4b0c1d7dff3bfd4035672a5086deec6f73979a49be73.js" as="script">
+
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_importmap_tags %>
+    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
```

Comment:
- [ ] When I run the app I get an error and we can test the project, please kindly fix that 🙏🏾 

<img width="735" alt="Screenshot 2023-06-17 at 15 09 25" src="https://github.com/Becky449/hello-rails-react/assets/74501523/d9c92c2c-a913-4523-b277-18ec0fa24dcf">